vm/compiler/src/ledger/get.rs

```diff
@@ -75,6 +75,20 @@ impl<N: Network, B: BlockStorage<N>, P: ProgramStorage<N>> Ledger<N, B, P> {
         }
     }
 
+    /// Returns the transaction for the given transaction id.
+    pub fn get_transaction(&self, transaction_id_string: &str) -> Result<Transaction<N>> {
```

Comment:
```suggestion
    pub fn get_transaction(&self, transaction_id: N::TransactionID) -> Result<Transaction<N>> {
```

This should be ` transaction_id: N::TransactionID` instead of a `&str`. The subsequent functions that call this can parse a string beforehand.

Reply:
Changed here [c6e67b7](https://github.com/AleoHQ/snarkVM/pull/996/commits/c6e67b7b363c5517adabff993563d293058ea3c0)

Reply:
Thanks!